From 2003638ee2dba31a4768d4296b254702af78a65e Mon Sep 17 00:00:00 2001
From: Dmitry Bulakh <dima@pkims.ru>
Date: Tue, 20 Oct 2020 08:07:26 +0300
Subject: [PATCH] update

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index a2827a0..9913c28 100644
--- a/.gitignore
+++ b/.gitignore
@@ -36,3 +36,4 @@ bin/LayoutTools.ilk
 bin/LayoutTools.pdb
 bin/tests/Prop2_v1.gds
 bin/tests/Prop2_v1.png
+build/msvs2015/.vs/
-- 
2.9.2.windows.1

